Neos.Flow/composer.json

```
@@ -35,6 +35,7 @@
         "psr/http-server-handler": "^1.0",
 
         "ramsey/uuid": "^3.0 || ^4.0",
+        "laminas/laminas-code": "~3.4.1|^4.0",
```

Same result, but consistent with what we usually do. Also the official syntax according to https://getcomposer.org/doc/articles/versions.md#version-range

```suggestion
        "laminas/laminas-code": "~3.4.1 || ^4.0",
```